common.go

```diff
@@ -141,6 +142,12 @@ func applyPathModifiers(path string, modifiers []string) string {
 		if modifier == "basename" {
 			replacement = basenamePtn.ReplaceAllString(replacement, "")
 		}
+
+		// If the |-separated part is "dirname", then extract only
+		// the directory of the path.
+		if modifier == "dirname" {
```

Comment:
Great, but I think we could improve the code here by using a [switch statement](https://tour.golang.org/flowcontrol/9), so that not more code is executed than necessary.

Reply:
Thank you for the advice. I changed to a switch statement here.